pkg/lint/rules/values.go

```diff
@@ -41,7 +41,25 @@ func ValuesWithOverrides(linter *support.Linter, valueOverrides map[string]inter
 		return
 	}
 
-	linter.RunLinterRule(support.ErrorSev, file, validateValuesFile(vf, valueOverrides))
+	linter.RunLinterRule(support.ErrorSev, file, validateValuesFile(vf, valueOverrides, false))
+}
+
+// ValuesWithOverridesWithSkipSchemaValidation tests the values.yaml file.
+//
+// If a schema is present in the chart, values are tested against that. Otherwise,
+// they are only tested for well-formedness.
+//
+// If additional values are supplied, they are coalesced into the values in values.yaml.
+func ValuesWithOverridesWithSkipSchemaValidation(linter *support.Linter, values map[string]interface{}, skipSchemaValidation bool) {
```

Comment:
This function duplicates the lint rules in `ValuesWithOverrides`, and as such these are going to be run twice, and report issues twice to the user

That said, now we are targeting Helm 4, we don't need to maintain backward compatiblity we function signatures, and can simply add the new `skipSchemaValidation` parameter to `ValuesWithOverrides`

(this will preclude backporting this to Helm 3 in its current form)